### DISCUSSION TITLE
iNav to support out-of-the box presets for -8-10 inch props

### DISCUSSION THREAD

**Discussion by Ravid824** [iNavFlight/inav#6552] [Ideas]
Currently the out-of-the box presets for INAV works well for quads with 3-7" props.
The current quad 9" profile behavior is very very bad, easily can results with crashes.
The request is to extend the profiles support for 8-10" props (quad F450, F550 frams).
If I will use 7" profile on F450 quad 9" prop, do you think the behavior will be better?

**Comment by aa-calvo** [comment]
I agree, preset for large drones are pretty bad, there should at least be some documentation on tuning to help cope with larger drone

**Comment by DzikuVx** [comment]
Such a thing will happen only if someone with such a multirotor will prepare such a preset. From what I know, none of the devs have anything bigger than 7"

**Reply by Ravid824** [reply]
In iNAV 4.0 release note it is mentioned 
" Autotune no longer tunes P and I
There were often autotunes in iNav 3.0 which resulted in too high P and I. Also, D was not tuned at all. New default PIDs, that will work reasonably on all sizes of planes (#1390) have been added. This should give good results with an autotune, which can then be fine tuned to your plane."
Questions:
* It is not clear if Autotune is still available and works. The header say "Autotune no longer tunes P and I" while below 
  it says "This should give good results with an autotune". So what Autotune does?
* The new PID are for drone as well or just for plans?

Also, why PPM receivers were removed from version 4.0 . I need to update so many components to support serial !!!!!
There is a good reason for this?
David
